### ISSUE TITLE
Memory leak: global_params

### ISSUE THREAD

**Issue by jdukat** [tesseract-ocr/tesseract#607]
This code:

```
tesseract::ParamsVectors *GlobalParams() {
  static tesseract::ParamsVectors *global_params =
    new tesseract::ParamsVectors();
  return global_params;
}
```
is causing a memory leak. I don't see any means of removing it while tesseract engine is no longer needed in the application.

**Comment by stweil** [comment]
This was fixed in commit 53c572b4.

**Comment by jdukat** [comment]
Thanks!
